benchmark/scenarios/_rate_measurer.py

```diff
@@ -30,23 +36,25 @@ def request_sent(self):
         """
         self._sent += 1
 
-    def response_received(self, ignored):
+    def response_received(self, calltime):
         """
         Increase the number of received requests.
 
-        :param ignored: The result of a callback. This parameter is
-            not used.
+        :param float calltime: Time to perform call
```

Comment:
Please say "Time taken to perform call" to disambiguate from the other meaning of "When to perform the call"